README.md

```diff
@@ -1,27 +1,508 @@
 # React Changing State
 
-## Objectives
+## Overview
 
-1. Practicing updating state in React
-2. Practice passing state through props
-3. Describe how React rerenders on state changes
+React allows you to structure your application using small, reusable components,
+but React isn't really just about creating static pages, it's about building
+**react**ive user interfaces that enable you to coordinate complex user
+interactions.
 
-## Overview
+In this lesson we will be learning about how you can update the state of
+components. You are going to learn about the difference between `state` and
+`props` and what happens when state changes in your application occur.
+
+## State vs Props
+
+Managing state is hard. React allows you to encapsulate two different "kinds" of
+state: "internal" state that can only be mutated and accessed by the component
+that manages it (`state`) and "external" state that gets passed down to the
+component that renders it (`props`). This sounds way more complicated than it
+actually is, so let's take a step back and look at a simple example: A blog. The
+following examples are going to concentrate on a `Blog` component that renders
+multiple `BlogPost`s. Visually, this might look something like this:
+
+![Blog](./diagrams/overview.png)
+
+### Pure Components
```

Comment:
We can skip this stuff — pure components are talked about in Unit 6. Right now we're only concerned with having the students practice on getting used to `state`. I like the first part (up to but not including the 'The above component only consists of a render function...) part) so I think we can repurpose this and add it to the above section instead.